pkgs/development/python-modules/shapely/default.nix

```diff
@@ -58,6 +52,12 @@ buildPythonPackage rec {
 
   disabledTests = [
     "test_collection"
+
+    # FIXME(lf-): these logging tests are broken, which is definitely our
+    # fault. I've tried figuring out the cause and failed.
+    "test_error_handler_exception"
+    "test_error_handler"
+    "test_info_handler"
```

Comment:
Do you have the sandbox enabled on darwin? Because these tests pass fine for me on linux and darwin with the sandbox disabled.

Reply:
Can't remember. But we probably need them disabled given they break in one of the configurations.

Reply:
My point is we could probably downgrade these to a darwin-only disable, which is quite a common thing to do.